data/thehashmap.json

```diff
@@ -0,0 +1,85 @@
+{
+    "name": "Nikhil Kumar",
+    "type": "personal",
+    "bio": "Full-stack Web3 Developer | Open-Source Contributor | Blockchain | Ethereum | DeFi | 6⭐ @HackerRank | IIT (BHU) Varanasi",
+    "avatar": "https://github.com/thehashmap.png",
```

Comment:
```suggestion
```

GitHub Avatar is automatically picked up, the key is not needed.

Reply:
Okay. Thanks for the review!
Let me update it